src/EditorFeatures/CSharpTest/GenerateVariable/GenerateVariableTests.cs

```diff
@@ -22,6 +23,10 @@ public class GenerateVariableTests : AbstractCSharpDiagnosticProviderBasedUserDi
         internal override (DiagnosticAnalyzer, CodeFixProvider) CreateDiagnosticProviderAndFixer(Workspace workspace)
             => (null, new CSharpGenerateVariableCodeFixProvider());
 
+        private static readonly CSharpParseOptions WithPatternsEnabled = 
```

Comment:
>private [](start = 8, length = 7)

You don't need to do this any longer. The new pattern features are in C# 8, which is the default on this feature branch.

Reply:
thanks!